crates/simd-test-macro/src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub fn simd_test(
         "i686" | "x86_64" | "i586" => "is_x86_feature_detected",
         "arm" | "armv7" => "is_arm_feature_detected",
         "aarch64" => "is_aarch64_feature_detected",
+        maybe_riscv if maybe_riscv.starts_with("riscv32") || maybe_riscv.starts_with("riscv64") => {
```

Comment:
what are the kinds of `maybe_riscv` you expect to match?

Reply:
They are all the standard library compiled target supported by rust, i.e. riscv32i, riscv32imc, riscv32imac, riscv32gc, riscv64imac and riscv64gc. I expect to match all of these targets, for shorter code and forward compatible I used if match arm. @lu-zero

Reply:
if `riscv128` is a future option maybe you can use a `single starts_with`.

Reply:
@lu-zero Yes, maybe starts with "riscv" is sufficient here.